AdSecGH/Parameters/AdSecProfileGoo.cs

```diff
@@ -307,7 +307,7 @@ public class AdSecProfileGoo : GH_GeometricGoo<Polyline>, IGH_PreviewData {
     }
 
     public override string ToString() {
-      return "AdSec " + TypeName + " {" + Profile.Description() + "}";
+      return Profile.Description();
```

Comment:
@kpne I think that was to allow interop with GsaGH.. Can we live with that change?

Reply:
@tlmnrnhrdt  Yes